src/webhooks/pubsub/stalebot.ts

```diff
@@ -54,7 +54,7 @@ const staleStatusUpdater = async (
                 isPullRequest ? 'pull request' : 'issue'
               } has gone three weeks without activity. In another week, I will close it.
 
-But! If you comment or otherwise update it, I will reset the clock, and if you label it \`Status: Backlog\` or \`Status: In Progress\`, I will leave it alone ... forever!
+But! If you comment or otherwise update it, I will reset the clock, and if you remove the label \`Waiting for: Community\`, I will leave it alone ... forever!
```

Comment:
This comment was not entirely accurate before since the `Status: Backlog` and `Status: In Progress` logic didn't exist in eng-pipes. I held it off since it seemed silly to implement the logic then delete it a month later